src/lib/cookies.js

```diff
@@ -107,7 +107,21 @@ class CookieHandler extends CDP {
             await this.Network.deleteCookies(badCookie);
         }
         // Store cookies in the browser
-        await this.Network.setCookies({cookies: browserCookies});
+        const cleanedCookies = browserCookies.reduce((cookies, cookie) => {
+            if (cookie.name !== "") {
```

Comment:
The Hossein in my head says you should do this:
```suggestion
            if (cookie.name === "") 
              return cookies;
```